app/views/rpt/tournament_reports/show.html.haml

```diff
@@ -1,18 +1,17 @@
--#%h2= page_title
-%h2 U.S. Open Players Report
+%h2= @page_title
 
 - amnh = Attendee.model_name.human
-- xml_path = rpt_usopen_players_report_path + '.xml'
+- xml_path = rpt_tournament_reports_path + '.xml'
 
 = form_tag(xml_path, :method => :get) do
   %p.pull-right= submit_tag "Export"
 
 %p
   There are
-  = pluralize(@players.count, amnh.downcase)
+  = usgc_pluralize(@players.count, amnh.downcase)
```

Comment:
@neagle Is `usgc_pluralize` going to be an issue? This is something I missed on the merge.